sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -3419,6 +3419,39 @@ class Dataset[T] private[sql](
     } (Encoders.STRING)
   }
 
+  /**
+   * Returns the content of the Dataset as a Dataset of JSON strings.
+   * @since 3.0.1
```

Comment:
note: we cannot add a new feature in v3.0.x, so not `3.0.1` but `3.1.0`.

Reply:
ok!